haskell-language-server.cabal

```diff
@@ -497,6 +497,9 @@ executable haskell-language-server
 
   default-language: Haskell2010
   default-extensions: DataKinds, TypeOperators
+  
+  if impl(ghc >= 9.4)
+    build-depends: ghc-exactprint >= 1.6.1
```

Comment:
This doesn't seem correct. Surely the dependency is from another package, and that's what should be updated?

Reply:
OK, removed. 

Somewhere, something was pulling `ghc-exactprint-1.6.0`. I checked `ghcide`, and it was not it. I checked `hls-refactor-plugin`, and it had ` , ghc-exactprint < 1 || >= 1.4` - which should pull 1.6.1, but in some cases in my experience didn't (no idea why).

Reply:
Results show that the "wrong" works, and the "right" fails the tests. I'm leaving PR-that-works here.